spreadsheets/intermediate/charts-2/further-customization-1.md

```diff
@@ -0,0 +1,56 @@
+---
+author: Stefan-Stojanovic
+
+aspects:
+  - introduction
+
+type: normal
+
+category: how to
+
+---
+
+# Further customization 1
+
+---
+## Content
+
+Now that we know how to add more data, we can better visualize it by modifying our chart.
+
+The chart from the previous insight:
+
+![previous-chart](https://img.enkipro.com/a8f69242523d86c29148a4c3544b41eb.png)

```

Comment:
we can make these images better.
1. set the chart background to `None`
2. choose colours such that the charts is nicely displayed on both themes